README.md

```diff
@@ -14,25 +14,31 @@ Java idiomatic client for [Google Cloud Platform][cloud-platform] services.
 
 This client supports the following Google Cloud Platform services at a [Beta](#versioning) quality level:
 
--  [Google Cloud BigQuery](#google-cloud-bigquery-beta) (Beta)
+-  [BigQuery](#google-cloud-bigquery-beta) (Beta)
 -  [Stackdriver Logging](#stackdriver-logging-beta) (Beta - Not working on App Engine Standard)
--  [Google Cloud Datastore](#google-cloud-datastore-beta) (Beta)
--  [Google Cloud Storage](#google-cloud-storage-beta) (Beta)
--  [Cloud Spanner](#cloud-spanner-beta) (Beta)
--  [Google Cloud Translation](#google-translation-beta) (Beta)
--  Cloud Vision (Beta)
--  Cloud Natural Language (Beta)
+-  [Datastore](#google-cloud-datastore-beta) (Beta)
+-  [Storage](#google-cloud-storage-beta) (Beta)
+-  [Spanner](#cloud-spanner-beta) (Beta)
 
 This client supports the following Google Cloud Platform services at an [Alpha](#versioning) quality level:
 
--  [Google Cloud Compute](#google-cloud-compute-alpha) (Alpha)
--  [Google Cloud DNS](#google-cloud-dns-alpha) (Alpha)
--  [Google Cloud Pub/Sub](#google-cloud-pubsub-alpha) (Alpha - Not working on App Engine Standard)
--  [Google Cloud Resource Manager](#google-cloud-resource-manager-alpha) (Alpha)
+-  [Compute](#google-cloud-compute-alpha) (Alpha)
+-  [DNS](#google-cloud-dns-alpha) (Alpha)
+-  [Errorreporting](#google-cloud-errorreporting-alpha) (Alpha)
+-  [Language](#google-cloud-language-alpha) (Alpha)
+-  [Monitoring](#google-cloud-monitoring-alpha) (Alpha)
+-  [Pub/Sub](#google-cloud-pubsub-alpha) (Alpha - Not working on App Engine Standard)
+-  [Resource Manager](#google-cloud-resource-manager-alpha) (Alpha)
+-  [Speech](#google-cloud-speech-alpha) (Alpha)
+-  [Trace](#google-cloud-trace-alpha) (Alpha)
+-  [Translation](#google-translation-beta) (Alpha)
+-  [Vision](#google-cloud-vision-alpha) (Alpha)
```

Comment:
The names here shouldn't be shortened like this - they should be the official API names. For example, `Cloud Pub/Sub` (https://cloud.google.com/pubsub/).